### PR TITLE
ATv2 and MoSh: Enable nrf_cloud configs to auto send shadow data 

### PR DESCRIPTION
**Pull request by jayteemo**
ATv2:
Enable CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_UI and CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_FOTA.
MoSh:
Enable CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_UI in the MQTT and CoAP overlays.

Remove code from each application that sends the service info shadow update.
IRIS-6851

### CODE REVIEW

**Review by peknis — APPROVED** [review]
Approved with a minor nit.

File: samples/cellular/modem_shell/overlay-cloud_coap.conf
Comment (tokangas) [inline samples/cellular/modem_shell/overlay-cloud_coap.conf:23]: Currently, mosh only supports sending location data to the cloud. `CONFIG_NRF_CLOUD_ENABLE_SVC_INF_UI_RSRP` seems to be enabled by default, maybe that should be explicitly disabled in both CoAP and MQTT overlays.

Previously, the map card has been enabled only when location support is enabled in mosh, but I don't think that's too important.

**Review by gregersrygg — APPROVED** [review]

**Review by maxd-nordic — APPROVED** [review]
atv2 changes look good to me

File: samples/cellular/modem_shell/overlay-cloud_coap.conf
Comment (jayteemo) [reply samples/cellular/modem_shell/overlay-cloud_coap.conf:23]: sure, i will disable it

**Review by trantanen — APPROVED** [review]

**Review by trantanen — APPROVED** [review]
Fine for modem shell.
@tokangas is out of office. No need to wait for his approval.

**Review by jhirsi — APPROVED** [review]

**Pull request merged**
